sdcm/cluster.py

```diff
@@ -1434,6 +1466,47 @@ def get_console_screenshot(self):
         return ''
 
 
+    # Default value of murmur3_partitioner_ignore_msb_bits parameter is 12
+    def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits=12):
+        def _wait_for_resharding(timeout=300, reshard_found=False, start_search_from=0):
```

Comment:
Why _wait_for_resharding and not using the wait_for like all other places we have?
Maybe better to do like other places we did:
wait.wait_for(func=...)
Then, you can implement a func is_resharding or something like that.
Also, definition shouldn't be inside the "restart_node_with_resharding" func.

Reply:
@roydahan  Why we need it outside? Do are we going to use it in other place?

Reply:
Someone might want to check if there is resharding going on in the future.
why keeping it inside if it may be useful to other functions.